src/Core/tests/Benchmarks/Benchmarks/GridLayoutManagerBenchMarker.cs

```diff
@@ -177,7 +177,7 @@ static GridLength GridLengthFromString(string gridLength)
 				return new GridLength(double.Parse(gridLength), GridUnitType.Star);
 			}
 
-			if (gridLength.ToLower() == "auto")
+			if (gridLength.ToLowerInvariant() == "auto")
```

Comment:
Should this maybe be Equals with Comparison args?

Reply:
That would seem to be semantically more correct, though this being only a test, I don't mind either way.

Reply:
Ah, a test. I guess we don't really care then.

Reply:
It still needs to be functionally correct, which this change achieves (otherwise it could conceivably fail on some non-English-US culture in some odd way).

Reply:
The ones that were just tests, I just did Find/Replace to fix. I think it's ok to have non-optimal code like `if (gridLength.ToLowerInvariant() == "auto")` in tests.